packages/zowe-explorer/src/job/actions.ts

```diff
@@ -57,6 +58,38 @@ export async function downloadSpool(jobs: IZoweJobTreeNode[]): Promise<void> {
     }
 }
 
+/**
+ * Download all the spool content for the specified job.
+ *
+ * @param job The job to download the spool content from
+ */
+export async function downloadSingleSpool(nodes: IZoweJobTreeNode[], binary?: boolean): Promise<void> {
+    ZoweLogger.trace("job.actions.downloadSingleSpool called.");
+    try {
+        const dirUri = await Gui.showOpenDialog({
+            openLabel: localize("downloadSpool.select", "Select"),
+            canSelectFolders: true,
+            canSelectFiles: false,
+            canSelectMany: false,
+            defaultUri: getDefaultUri(),
+        });
+        if (dirUri !== undefined) {
+            for (const node of nodes) {
+                const spools = (await getSpoolFiles(node)).filter((spool: zowe.IJobFile) => matchSpool(spool, node));
+                for (const spool of spools) {
+                    await ZoweExplorerApiRegister.getJesApi(nodes[0].getProfile()).downloadSingleSpool({
```

Comment:
@zFernand0 Could we add the check for existence of new API with appropriate message for uses if the extender doesn't have it. Getting this error message now with extender that doesn't have it
![Screen Shot 2023-04-24 at 11 37 59 AM](https://user-images.githubusercontent.com/49491949/234046905-97f84550-8757-407d-8aa1-fac310e31371.png)

Reply:
Thanks for pointing this out 😿 
I keep forgetting to handle these cases for extenders.
Added a small message and a test for it 😋